devel/ddev/Portfile

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/ddev/ddev 1.23.5 v
+go.offline_build    no
+fetch.type          git
+
+categories          devel
+license             Apache-2
+maintainers         {@jlorenzetti} openmaintainer
+
+description         Local PHP development environment management tool
+long_description    DDEV is an open source tool for launching local PHP web development \
+                    environments in minutes. It helps developers create and manage \
+                    Docker-based development environments with support for multiple PHP \
+                    versions, web servers, databases, and other tools.
+
+homepage            https://ddev.com
+
+# Dependencies
+depends_build-append \
+                    port:gmake
+
+depends_run         port:mkcert
+
+# Build configuration
+build.env-append    CGO_ENABLED=0
+
+build.env-append    GOFLAGS=-mod=vendor\ -buildvcs=false
+
+build.cmd           ${prefix}/bin/gmake
+
+# Set build target based on architecture
+if {${build_arch} eq "x86_64"} {
+    set go_arch "amd64"
+} elseif {${build_arch} eq "arm64"} {
+    set go_arch "arm64"
+}
+
+build.target    darwin_${go_arch}
+
+destroot {
+    # Install binary
+    xinstall -m 755 ${worksrcpath}/.gotmp/bin/darwin_${go_arch}/ddev ${destroot}${prefix}/bin/
+    
+    # Generate shell completions
+    system -W ${worksrcpath} "git config --global --add safe.directory ${worksrcpath}"
```

Comment:
Is this `git config` command still necessary? Especially that it's making a `--global` config change... I think that's something we should avoid since it can change the external system environment outside of the build root where the port is being built.